README.md

```diff
@@ -17,7 +17,7 @@ First, [install Rust](https://www.rust-lang.org/tools/install).
 Then, install the required WasmEdge WASM runtime:
 
 ```sh
-curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install_v2.sh | bash
+curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install_v2.sh | bash -s -- --version=0.14.0
```

Comment:
We're also using this command in our build scripts, so we should take care of that.

<img width="1129" alt="image" src="https://github.com/user-attachments/assets/b686bdd5-c9c4-4460-8feb-8cab94de572a">

@kevinaboos Any additional recommendation or tip to make the changes in those places?

Reply:
Let's merge this since we have related reports of people trying to build it. We can catch up with the release script in a separate PR.

Reply:
PR #179 fixes this.